benchmark/test/benchmark/measure.clj

```diff
@@ -0,0 +1,68 @@
+(ns benchmark.test.measure
+  (:require [clojure.test :as t :refer [is deftest]]
+            [benchmark.measure :as b]))
+
+(def config {:config-name :all
+             :output-format "edn"
+             :data-types [:int :str]
+             :iterations 1, 
+             :data-found-opts :all
+             :tx-entity-counts [0 10]
+             :tag #{test} 
+             :query :all 
+             :db-entity-counts [0 10]})
+
+(deftest connection-test 
+  (let [measurements (b/get-measurements (assoc config :function :connection))]
+    (is (= (set (keys (:time (first measurements)))) 
```

Comment:
For the future tests: the value you are expecting is the first argument. Some testing tools prefer this order in order to tell expectation vs actual apart.